backplane/build/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Pre-install packages
+FROM quay.io/centos/centos:latest
```

Comment:
We shouldn't use centos image. Should use any image which is under https://catalog.redhat.com/software/containers/explore and is supported by Redhat.